doc/fta-construction-algorithm.md

```diff
@@ -39,11 +39,21 @@ BEGIN
 END
 ```
 
-## Example 1
+## Example 1 - Or Behaviour
 Demonstrates transformation of OR behavior.
 ![convert-require-imapair-behavior-model-to-fault-tree](https://user-images.githubusercontent.com/18463762/134146813-8a257931-26bb-4350-b954-c29ac7601c9c.png)
 
-## Example 2
+## Example 2 - And Behaviour
 Demonstrates transformation of AND behavior.
 ![convert-require-imapair-behavior-model-to-fault-tree-example-2](https://user-images.githubusercontent.com/18463762/137472141-065769d1-8f6c-4267-ac5e-4fc85c16a0c4.png)
 
+## Example 3 - Live Demo
+### FTA EXamples
+This example is a study by [1]. The analyzed system in this example is aircraft fuel system, including its subsystems (fuel transportation, fuel supplying and monitoring subsystem).  The following schema shows a fault tree that is created by this tool where circles represent root contributory factors, rectangles represent intermediate events, Boolean operators “AND” and “OR” are depicted with dedicated objects between intermediate events and contributory factors or other intermediate events.
```

Comment:
This page is about FTA transformation or more generally about FTA construction ... But here I did not find out why you did. 

So did you generate it from a system ? From what system ?
I would show:
1) picture of the system (+ add link)
2) picture of the generated FTA  (+ add link, more concretely https://kbss.felk.cvut.cz/fta-fmea-demo/fta/instance-1287330450)
3) if it is true, described what did you have to do to achieve 2) ... did you have to add probabilities ?


"he following schema shows a fault tree that is created by this tool where circles represent root contributory factors, rectangles represent intermediate events, Boolean operators “AND” and “OR” are depicted with dedicated objects between intermediate events and contributory factors or other intermediate events."
-- good sentence, but belong to the beginning of this page !!! Because we already showed 2 pictures with this notation.  I would just reformulate "circles" => "circles or ovals"

Reply:
- The tool can only generate fta from a function.  It can't generate fta for the whole system. it generates a fault tree for one function in a component. Therefore, I crated fta from tool not generated.
- In the tool, every component can be connected to only one component, so we can't crate this example by our tool.

Reply:
and regarding probabilities, we have to add them.

Reply:
> * The tool can only generate FTA from a function.  It can't generate FTA for the whole system. it generates a fault tree for one function in a component. Therefore, I created FTA from tool not generated.

- So isn't it just a function composition problem? Can't we just have a function "to behave correctly for all use-cases"  and use function composition for the case?

@Matthew-Kulich @kostobog @lalisand  What do you think about this? 

> * In the tool, every component can be connected to only one component, so we can't crate this example by our tool.

@Matthew-Kulich is this true?

Reply:
Yes, it is true

Reply:
@blcham @lalisand even with this hint we can't represent a paper example by using this tool. because many components are connected to more than one component (e.g., oil supplying tanks, pressure pump 1,2, etc).
but I added a link to the part of the system that is created by the tool

Reply:
@lalisand yes exactly. that's why it is better to not represent it by our tool. right?

Reply:
@ahmadjana not sure what is the goal to represent it in our tool - if it is a visualization of the system, then this is not a good idea; if this is some database we show, then it might be useful

Reply:
**The goal here is to represent the system's reliability by a generated FTA tree** -- moreover, show the tool's limitations or specifics related to the task.

Thus, I suggest [1] keeping this picture of functional dependencies here while [2] representing this system in our tool using part-of relations. [3] Explain that the first picture is about functional dependencies combined with part of relations ... and that in the tool, it can be represented as such + show here the picture ...  [4] Then add one function to combine all critical functions + explain that this is a limitation of our tool and [5] generate FTA representing system's reliability + add the probabilities of leaf nodes + show here the picture.

@ahmadjana @lalisand  If you don't consider it meaningful, close the ticket.

Reply:
@blcham ok, but if we talk about the reliability paper, I do not entirely get how your suggestions match with it. But it sounds reasonable to me. @ahmadjana @kostobog - your opinion?

Reply:
@lalisand I think it is enough what we have now and we can close the ticket

Reply:
@kostobog ?

Reply:
**Clarification of the system modeling capabilities of the tool:**
- the tool represent part-whole relationships where each part is allowed to have only one parent. 
- the tool can represent behavioral relationships:
  - behavioral part-whole relationships (again behavioral parts are allowed to be a part of one whole) - this includes functions composition of other functions and failure mode composition of other failure modes.
  - behavioral dependency (many to many relation) - this includes functions requiring other functions as well as failure modes requiring other failure modes.
  - behavioral "conflicts" - allows to model that a function is impaired by a failure mode.

The system diagram shows only part whole relationships between components. Behavioral relationships can be seen and managed only in the component form.

Reply:
@kostobog can you incorporate it in the PR ?

Reply:
@blcham 
I reviewed and rewritten the description of the system modeling and fault tree construction. The original description was missing details and had some mistakes.

A new issue was created to resolve broken links to fta-fmea-demo #158
- This task  requires to fix issues with demo deployment.